scripts/download-wda-sim.mjs

```diff
@@ -1,4 +1,4 @@
-import {fs, logger, zip, net, node} from 'appium/support.js';
+import {fs, logger, zip, net, node} from '@appium/support';
```

Comment:
we should also verify that this module is explicitly added to package dependencies